hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/ClientClusterServiceImpl.java

```diff
@@ -208,69 +174,50 @@ public void waitInitialMemberListFetched() {
         }
     }
 
-    public void clearMemberListVersion() {
+    public void onClusterRestart() {
```

Comment:
I think the names `onClusterRestart` and `onClusterChange` 's confusing. Same methods also exist at HazelcastClientInstanceImpl.